static/app/components/tabs/tabPanels.tsx

```diff
@@ -0,0 +1,99 @@
+import {useContext, useRef} from 'react';
+import styled from '@emotion/styled';
+import {AriaTabPanelProps, useTabPanel} from '@react-aria/tabs';
+import {useCollection} from '@react-stately/collections';
+import {ListCollection} from '@react-stately/list';
+import {TabListState} from '@react-stately/tabs';
+import {CollectionBase, Node, Orientation} from '@react-types/shared';
+
+import space from 'sentry/styles/space';
+
+import {TabsContext} from './index';
+
+interface TabPanelsProps<T> extends AriaTabPanelProps, CollectionBase<T> {
+  className?: string;
+}
+
+/**
+ * To be used as a direct child of the <Tabs /> component. See example usage
+ * in tabs.stories.js
+ */
+export function TabPanels<T extends object>(props: TabPanelsProps<T>) {
+  const {
+    rootProps: {orientation, items},
+    tabListState,
+  } = useContext(TabsContext);
+
+  // Parse child tab panels from props and identify the selected panel
+  const factory = (nodes: Iterable<Node<T>>) => new ListCollection(nodes);
```

Comment:
Feels like this should be defined outside of the component

Reply:
True – moved it outside in https://github.com/getsentry/sentry/pull/38886/commits/bb12b648ae479435ea74c5c76ce976dfb89150ad